pkg/service/targetgroupsvc/create.go

```diff
@@ -0,0 +1,36 @@
+package targetgroupsvc
+
+import (
+	"context"
+	"strings"
+
+	"github.com/common-fate/common-fate/pkg/targetgroup"
+	"github.com/common-fate/common-fate/pkg/types"
+	"go.uber.org/zap"
+)
+
+func (s *Service) CreateTargetGroup(ctx context.Context, req types.CreateTargetGroupRequest) (*targetgroup.TargetGroup, error) {
+	log := zap.S()
+
+	//look up target schema for the provider version
+
+	splitKey := strings.Split(req.TargetSchema, "/")
+
+	resp, err := s.ProviderRegistryClient.GetProviderWithResponse(ctx, splitKey[0], splitKey[1], splitKey[2])
```

Comment:
You will need to validate here that the length of the split target is what you expect